Docs - updates for PL/Container 3 beta

dyozie opened the pull request.

lenapivotal commented:
> * if the user wants to use pl/container 3 beta, i think that they must uninstall pl/container 2.x, and remove the images.  if they remove the images, then maybe they also need to review the runtimes?  this is important, because it looks to me like the previous and beta R images have the same name.  (not sure if installing the beta image over the previous will just update it.)

So, we have a section about upgrading PL/Container, staging app here: 
http://docs-lena-plcontainer.cfapps.io/7-0/analytics/pl_container.html#upgrade_plcontainer

Does this process not work for 3 beta?

dyozie commented:
I'm going to run through that today to see if anything special needs to be
done to overlay 3.0 beta.

On Thu, Mar 19, 2020 at 10:57 AM Lena Hunter <notifications@github.com>
wrote:

>
>    - if the user wants to use pl/container 3 beta, i think that they must
>    uninstall pl/container 2.x, and remove the images. if they remove the
>    images, then maybe they also need to review the runtimes? this is
>    important, because it looks to me like the previous and beta R images have
>    the same name. (not sure if installing the beta image over the previous
>    will just update it.)
>
> So, we have a section about upgrading PL/Container, staging app here:
>
> http://docs-lena-plcontainer.cfapps.io/7-0/analytics/pl_container.html#upgrade_plcontainer
> <https://urldefense.proofpoint.com/v2/url?u=http-3A__docs-2Dlena-2Dplcontainer.cfapps.io_7-2D0_analytics_pl-5Fcontainer.html-23upgrade-5Fplcontainer&d=DwMCaQ&c=lnl9vOaLMzsy2niBC8-h_K-7QJuNJEsFrzdndhuJ3Sw&r=m377qB3FLyOisKE6Va0Gg0pgOJmsnyyjjIQ_CgtFEEo&m=_dFP7Zi6w9gWqEYmwjIhQGwlfChbYeDmxkzTZImpkZk&s=LsQEkCiyaQZma2UPFolNz2tUj-rH_8b1pKRHEZfAj3o&e=>
>
> Does this process not work for 3 beta?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/greenplum-db/gpdb/pull/9773#issuecomment-601328177>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABLOSVUSDE4ZDGXO7KTMBFTRIJMJFANCNFSM4LO5ZZMQ>
> .
>

dyozie commented:
Thanks @lisakowen   I updated the docs to account for uninstalling existing pl/container software before installing the Beta.  Also updated the uninstall to account for Beta, and qualified the R image as beta.  I'm not sure what you had in mind re: italicizing the beta stuff.  I think each of the added instructions for Beta are pretty clear that they're only for Beta, so shouldn't be confusing in that regard.
